src/Bicep.LangServer/Telemetry/TelemetryConstants.cs

```diff
@@ -9,7 +9,7 @@ public static class TelemetryConstants
 
         public static class EventNames
         {
-            public const string LinterStateOnBicepFileOpen = nameof(LinterStateOnBicepFileOpen);
+            public const string LinterRuleStateOnBicepFileOpen = "linter/rulestateonopen";
```

Comment:
[nit]: It might be worth moving this to the bottom, so all linter event names stay together. Also, is it by design that the linter rule names are all in lowercase?

Reply:
Sure, updated here:
https://github.com/Azure/bicep/commit/a479c1b9b79e89543443adcf3d0935d034074a57

Rule names are all in lower case to help ease querying. The names get lowercased before they are stored. So doing it upfront here will avoid confusion while querying.

Reply:
Makes sense.